src/codegen/codegen.test.ts

```diff
@@ -210,9 +211,7 @@ describe('Code generation', () => {
 
       const expectedResult = await prettify(`
         const FILES = {
-          users: new SharedArray("users", () => {
-            return Papa.parse(open("../Data/users.csv"), { header: true }).data;
-          }),
+          users: await csv.parse(await fs.open("../Data/users.csv"), { asObjects: true }),
 
           products: new SharedArray("products", () => {
             const data = JSON.parse(open("../Data/products.json"));
```

Comment:
Feels a bit odd to have both "global" `open` and `open` from `k6/experimental/fs`